tests/test_gui.py

```diff
@@ -24,9 +25,17 @@
 
 @pytest.fixture(scope="session")
 def context():
-    # A task in Big Buck Bunny which we're going to use
-    # for the current context.
-    return {"type": "Project", "id": 65}
+    # Get credentials from TK_TOOLCHAIN env vars
+    sg = shotgun_api3.Shotgun(
```

Comment:
Please use this method to get a Toolkit user and then call create_sg_connection instead.
https://github.com/shotgunsoftware/tk-toolchain/blob/master/tk_toolchain/authentication.py#L57

Reply:
```
from tk_toolchain.authentication import get_toolkit_user
sg = get_toolkit_user().create_sg_connection()
```